helpme/tests.py

```diff
@@ -67,7 +69,7 @@ def test_create_ticket(self):
         ticket = Ticket.objects.get(subject="Test subject")
         self.assertEqual(ticket.user, self.support)
         self.assertQuerysetEqual(ticket.teams.all(), ['<Team: Support>'])
-        self.assertEqual(ticket.user_meta, {'Device': 'Other', 'Browser': 'Other ', 'IP Address': '127.0.0.1', 'Mobile/Tablet/PC': 'Unknown', 'Operating System': 'Other '})
+        self.assertEqual(ticket.user_meta, {'Device': 'Other', 'Browser': {'family': 'Other', 'version': ''}, 'IP Address': '127.0.0.1', 'Mobile/Tablet/PC': 'Unknown', 'Operating System': {'family': 'Other', 'version': ''}})
```

Comment:
the keys should be all lowercase with optional underscore and should not contain spaces or slashes.